ext/osx/postflight.erb

```diff
@@ -1,109 +0,0 @@
-#!/bin/bash
```

Comment:
I think these might be used when building dmg files here: https://github.com/puppetlabs/packaging/blob/65185922cbe33f94c16296ba825fe390c4573801/tasks/apple.rake#L61

Reply:
But there's also https://github.com/puppetlabs/vanagon/blob/main/lib/vanagon/platform/osx.rb, so I say remove these and see what breaks

Reply:
I thought this was also superseded by https://github.com/puppetlabs/vanagon/blob/master/lib/vanagon/platform/osx.rb but seems safer to not remove it 😅 .

Reply:
![Roger Roger](https://media1.tenor.com/images/b30ea4f0bb2562e749d00e846106eb82/tenor.gif?itemid=5912820)